### PR TITLE
Removed details and size from data discovery page

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Comment by fightingtexasaggie** [comment]
I added a commit that alternates row colors, but I don't really care for my choice of color. Feel free to change the color in data-discovery.css to something better, (or revert my commit).

**Comment by mickel1138** [comment]
Plain css striped is neater: https://www.w3schools.com/css/tryit.asp?filename=trycss_table_striped

File: src/Pelagos/Bundle/AppBundle/Resources/public/css/data-discovery.css [hunk 1]
```diff
@@ -140,13 +140,15 @@ input.ui-button {
     border-top-right-radius: 4px;
 }
 
+.datasets tr:nth-child(odd) {
```
Comment (mickel1138) [inline src/Pelagos/Bundle/AppBundle/Resources/public/css/data-discovery.css]: Odd or Even?

Comment (fightingtexasaggie) [reply src/Pelagos/Bundle/AppBundle/Resources/public/css/data-discovery.css]: Coloring the odds so the first row gets colored

**Pull request merged**